jitpack.yml

```diff
@@ -0,0 +1,9 @@
+jdk:
+  - oraclejdk8
+install:
+   - set -exo pipefail
+   - ant
+   - bin/buck build buck
```

Comment:
nit: you can avoid having to use `buck targets --show-output` and use `buck build --show-output` instead.